recipes/qarrayrun/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "qarrayrun" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "80441f5a0cf27e3ba8805ced549f27c176f43d356e6d783c33ef643585a106c5"
```

Could you please remove the quotes from the ``sha256``?
```suggestion
  sha256: 80441f5a0cf27e3ba8805ced549f27c176f43d356e6d783c33ef643585a106c5
```

Related to https://github.com/conda/conda-build/pull/3720

---

Sure, done